docs/tutorials/python/linear-regression.md

```diff
@@ -11,6 +11,13 @@ import mxnet as mx
 import numpy as np
 ```
 
+## Prerequisites
+1. [MXNet](http://mxnet.io/get_started/install.html), [Graphviz](../prerequisites/install-graphviz.html).
+2. Jupyter Notebook and Graphviz python packages.
+   ```
+   pip install jupyter graphviz
```

Comment:
If we still need to pip install graphviz, can you move it to the main page. Visualizing the graph is a pretty common usecase and can go on the main installation page.